code/modules/research/designs/circuits.dm

```diff
@@ -467,14 +467,14 @@ CIRCUITS BELOW
 	req_tech = list(TECH_DATA = 4, TECH_COMBAT = 2)
 	build_path = /obj/item/weapon/circuitboard/mecha/gygax/targeting
 	sort_string = "NAACC"
-/*							//Uncomment me to allow Serenity construction
+
```

Comment:
Since these are editing Polaris files that we got these from, I think that, IIRC about coding standards, the snippet `//Vorestation edit` should be added in to this.